sensor_msgs/PointCloud2[] clouds
---
int16[] classes_of_parts
